### ISSUE TITLE
Error migrating "All" forms from Freeform

### ISSUE THREAD

**Issue by jdsdev** [verbb/formie#567]
**Description**

Selecting "All" when migration forms from Freeform causes the migration to fail.

This occurs because `$forms` expects `$formIds` to be an array, and selecting "All" returns the string "*".

**Additional info**

- Plugin version: 1.4.15

**Comment by engram-design** [comment]
Ah, I forgot Freeform forms are handled differently (as opposed to Sprout Forms). Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.15",
  "...": "..."
}
```

Then run `composer update`.

**Comment by jdsdev** [comment]
That seems to have done the trick, thanks for the quick fix!

**Comment by engram-design** [comment]
Fixed in [1.4.16](https://github.com/verbb/formie/releases/tag/1.4.16)
